app/views/static/participate/_campaigns.html.erb

```diff
@@ -52,7 +51,7 @@
 
     <div class="wrap row">
       <div class="c-pagination col-xs-12 col-sm-6 col-sm-offset-3">
-        <!-- Implements the paginations once we get the campaigns (@campagins) -->
+        <!-- Implements the pagination once we get the campaigns (@campaigns) -->
         <% if @campaigns %>
```

Comment:
this will be removed, yes. but we will do it separately, as the whole thing needs a rework to work with the campaigns API